mir-module/src/main/resources/META-INF/resources/content/publish/index.xml

```diff
@@ -97,15 +90,16 @@
           action="../../editor/editor-dynamic.xed">
 
 
-
-      <h2>Publication</h2>
-      <h3> Set publication type </h3>
       <div class="row">
-        <div class="col-sm-9 offset-sm-3 col-md-7 offset-md-3 col-lg-5 offset-lg-2">
+        <div class="col-lg-7 col-sm-12">
+          <h2>Publication</h2>
           <p>
-            Please select the type of the publication.
-            Afterwards you will be directed to the appropriate form.
+            Please select the type of the publication. Afterwards you will be directed
+            to the appropriate form. If record already have an identifier you can enter one.
```

Comment:
wording?  suggestion: If you know an identifier for the publication record you may enter it here. It will be used to look up and reuse metadata from external databases.